src/lib/churn.ts

```diff
@@ -90,22 +77,45 @@ function assertIsGitRootDirectory(directory: string): void {
   }
 }
 
-function buildGitLogCommand(options: Options): string {
+function buildGitLogCommand(options: Options, isWindows: boolean): string {
   return [
-    [
-      "git",
-      `-C ${options.directory}`,
-      `log`,
-      `--follow`,
-      `--format=''`,
-      `--name-only`,
-      options.since ? `--since="${options.since}"` : "",
-      "'*'",
-    ]
-      .filter((s) => s.length > 0)
-      .join(" "),
-    "sort",
-    // --count might not be supported by all OS
-    "uniq -c",
-  ].join(" | ");
+    "git",
+    `-C ${options.directory}`,
+    `log`,
+    `--follow`,
+
+    // Windows CMD handle quotes differently than linux, this is why we should put empty string as said in:
+    // https://github.com/git-for-windows/git/issues/3131
```

Comment:
> Nice! 
> 
> 
> 
> I really like leaving this kind of comment that says "why" something is there.

Thank you!

> The best approach would be to have a test somewhere but testing for platform-specific code is always tricky, so that's good enough :+1: 

I see you use Travis.CI if you want maybe we can run specific tests based on the platform

Reply:
> I see you use Travis.CI if you want maybe we can run specific tests based on the platform.
> 
> Or maybe something even better is to run E2E tests on both platforms.
> 
> The way we could test that is by:
> 
>     1. Install the package globally per test - `npm install -g` will install the current package globally.
>     2. Use `child_process` and match the output to a snapshot (use already known repository with already known results)

Thanks for the insight :+1: 

In theory, I would agree. However I'd rather fix the bug first (and merge this PR) and decide on how to improve the testing strategy later since I'm not super happy with how I've done it :sweat_smile:

Reply:
@simonrenoult fair enough, so I guess it ready for merging 🚀